index.html

```diff
@@ -6,8 +6,14 @@
   </head>
   <body>
     <h1>Hello World!</h1>
+    <!-- All of the Node.js APIs are available in this renderer process. -->
     We are using node <script>document.write(process.versions.node)</script>,
     Chromium <script>document.write(process.versions.chrome)</script>,
     and Electron <script>document.write(process.versions.electron)</script>.
   </body>
+
+  <script>
+    // You can also require other files to run in this process
+    require('./renderer.js')
```

Comment:
Super minor, but the `.js` isn't required, if you think it makes it clearer I'd say leave it.